src/Mutation.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Softonic\GraphQL;
+
+use Softonic\GraphQL\Config\MutationConfigObject;
+use Softonic\GraphQL\Mutation\Item;
+use Softonic\GraphQL\Query\Item as QueryItem;
+
+class Mutation
+{
+    private $mutation;
```

Comment:
It's a little strange to have a Mutation class with a property that it's a mutation

Reply:
We're still discussing what a mutation should be, an array with all the variables of a mutation or just one of them.

Reply:
At the end ill be an item with all the variables.